rebar.lock

```diff
@@ -1,28 +1,26 @@
 {"1.1.0",
-[{<<"bbmustache">>,{pkg,<<"bbmustache">>,<<"1.6.0">>},0},
- {<<"certifi">>,{pkg,<<"certifi">>,<<"2.3.1">>},0},
+[{<<"bbmustache">>,{pkg,<<"bbmustache">>,<<"1.6.1">>},0},
+ {<<"certifi">>,{pkg,<<"certifi">>,<<"2.5.1">>},0},
  {<<"cf">>,{pkg,<<"cf">>,<<"0.2.2">>},0},
- {<<"cth_readable">>,{pkg,<<"cth_readable">>,<<"1.4.3">>},0},
+ {<<"cth_readable">>,{pkg,<<"cth_readable">>,<<"1.4.4">>},0},
  {<<"erlware_commons">>,{pkg,<<"erlware_commons">>,<<"1.3.1">>},0},
  {<<"eunit_formatters">>,{pkg,<<"eunit_formatters">>,<<"0.5.0">>},0},
  {<<"getopt">>,{pkg,<<"getopt">>,<<"1.0.1">>},0},
  {<<"parse_trans">>,{pkg,<<"parse_trans">>,<<"3.3.0">>},0},
- {<<"providers">>,{pkg,<<"providers">>,<<"1.7.0">>},0},
- {<<"relx">>,
-  {git,"https://github.com/aeternity/relx.git",
-       {ref,"ebdcebf135dfe03bec3801e1e1fba0ab1e1b36d0"}},
-  0},
- {<<"ssl_verify_fun">>,{pkg,<<"ssl_verify_fun">>,<<"1.1.3">>},0}]}.
+ {<<"providers">>,{pkg,<<"providers">>,<<"1.8.1">>},0},
+ {<<"relx">>,{pkg,<<"relx">>,<<"3.32.1">>},0},
```

Comment:
https://github.com/aeternity/relx/pull/5 will bring the https://github.com/aeternity/relx/tree/integration in line with upstream tag https://github.com/erlware/relx/tree/v3.32.1 so no need to point to the `aeternity/relx` fork here.